examples/simple.md

```diff
@@ -0,0 +1,49 @@
+Let's say you have the following Rust binary called `simple`:
+
+```rust
+use std::env;
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    if args.len() == 2 {
+        println!("Hello {}!", args[1]);
+    } else {
+        eprintln!("Must supply exactly one argument.");
+        std::process::exit(1);
+    }
+}
+```
```

Comment:
I'm trying to weigh having this file stand on its own and being able to copy/paste with README and single-sourcing `example.rs` by having most of this content moved to `lib.rs` with that "including" `sample.rs` and `sample.md` like in https://github.com/clap-rs/clap/blob/master/src/_tutorial.rs

I lean towards the latter as I also think I'm leaning towards keeping the markdown file simple.

Reply:
> Ok, this should be ready for another round of review.

Looks like this still needs to be responded  to or resolved.

Reply:
I guess I was trying to keep one contained file that was actually a trycmd test. If `simple.md` becomes just the ```console part, then you lose the ability to see a real md file. Things become magical again and hard to understand.

I removed the simple.rs copy and updated the quote at the bottom to explain it.